docs/wip/data-fetching.md

```diff
@@ -1,8 +1,58 @@
-## Gatsby config queries
-
 ## How to fetch data for your pages
 
+One of the main benefits of Gatsby is its hybrid nature capable of static and dynamic data fetching, which means that is possible to fetch data in build time as well in runtime on client-side. The choice of which approach to use depends on the nature of the data and its context, in the section below we will explain the difference with examples.
+
+*If you want to read more about this topic, check the Data Fetching Gatsby documentation.*
+
 ### Static data
+
+`Static data` is the content that does not change frequently or per user interaction.
+Considering that, you can use the fetching during building time for data that don't change very often, making the storefront loading faster.
+
+For example, a product's page can be a data retrieve static since the description and attributes of a product rarely chages. This means that this data can be fetch during build time and when the user access the products page the data is already loaded.
```

Comment:
```suggestion
Static data is the content that does not change frequently or per user interaction. We recommend using static rendering when the content doesn't change very often, and when you want to guarantee the fastest possible loading time.

Take the following example: imagine you have a product page, whose descriptions, attributes, and photos rarely change. In this case, you can benefit from retrieving fast static data during build time so that, when a customer accesses this product page, the data is almost instantly loaded.

>ℹ️ *Notice that, on the same page, you can have both static and dynamic data. For example, you can have dynamic data for your product price.*
```